src/pages/homeCollection/kitsReceipt.js

```diff
@@ -212,6 +211,30 @@ const storePackageReceipt = async (data) => {
       enableCollectionCheckBox();
       document.getElementById("packageCondition").setAttribute("data-selected","[]");
     }
+    
+    const returnedPtInfo = await response.json().then(data => { return data.response })
```

Comment:
there seems to be duplicated code with line 115. Should we create a common function to prepare data.